CONTRIBUTING.md

```diff
@@ -12,9 +12,9 @@ For more complex conversations, use the [discussions](https://github.com/unionla
 
 We evaluate the need for a PR based on:
 
-1. Severity of the issue (bug or feature request).
-2. Maintainability: will this become a burden for little gain, or add value?
-3. Can the core team understand the code additions being made, and maintain them, or will they rely on you in the future?
+1. Severity of the issue (bug or feature request),
+1. Maintainability: will this become a burden for little gain, or add value?
+1. Can the core team understand the code additions being made, and maintain them, or will they rely on you in the future?
```

Comment:
what lol

Reply:
yes

Reply:
the more you know

Reply:
I'm in doubt regarding this. I mostly rawdog .md files in the terminal rather than looking at the rendered output

Reply:
On the other hand, it does make diffs much nicer